account/src/main/java/com/user/account/services/serviceImplementation/AccountServiceClass.java

```diff
@@ -0,0 +1,27 @@
+package com.user.account.services.serviceImplementation;
+
+import com.user.account.advice.CustomExceptions;
+import com.user.account.message.DefaultMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.user.account.entity.User;
+import com.user.account.repositories.UserRepository;
+
+@Service
+public class AccountServiceClass implements AccountService {
+
+    @Autowired
+    private UserRepository userRepository;
+    
+    @Override
+    public User updateUser(User user) {
+        return  userRepository.save(user);
```

Comment:
[Minor] you've double space after `return`

Reply:
Remove a single space after return